render/src/pixel_bender.rs

```diff
@@ -272,7 +272,9 @@ pub enum PixelBenderShaderArgument<'a> {
     },
 }
 
-/// An image input. This accepts both an owned BitmapHandle,
+/// An image input.
+///
+/// This accepts both an owned BitmapHandle,
 /// and a borrowed texture (used when applying a filter to
 /// a texture that we don't have ownership of, and therefore
 /// cannot construct a BitmapHandle for).
```

Comment:
nit: I probably would have re-wrapped this neatly into 3 lines, but oh well...

Reply:
fixed